app/modules/asset_groups/models.py

```diff
@@ -960,10 +965,21 @@ def send_detection_to_sage(self, model):
                 }
                 # This is necessary because we can only mark self as modified if
```

Comment:
The code moved but the comment was left dangling and is not unclear.

Reply:
Removed